src/components/navigation-dropdown/navigation-dropdown.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ControlSelect from '@mapbox/react-control-select';
+// import { routeToPrefixed } from '@mapbox/batfish/modules/route-to';
+
+class NavigationDropdown extends React.Component {
+  render() {
+    const { props } = this;
+    const optionsArray = props.dropdownOptions.map(option => {
+      return {
+        label: option.title,
+        value: option.path
+      };
+    });
+    const currentOption = optionsArray.filter(option => {
+      return props.currentPath === option.value;
+    });
+
+    return (
+      <div className="py24 px24 flex-parent flex-parent--space-between-main flex-parent--center-cross">
```

Comment:
Are these flex classes necessary? Looks like there's only one flex-child?

Reply:
Must be leftover from an `earlier attempt`.